### PR TITLE
Use fragments from OCA for community addon. Add readme for non-community addon.

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
@xavier-bouquiaux @rousseldenis @acsonefho @Cedric-Pigeon

**Review by acsonefho — APPROVED** [review]

**Comment by rousseldenis** [comment]
@adrienpeiffer Could you check tests ?

File: bobtemplates/odoo/hooks.py [hunk 1]
```diff
@@ -147,29 +167,61 @@ def post_render_model(configurator):
...
+
 def post_render_addon(configurator):
     variables = configurator.variables
-    if variables["addon.oca"]:
-        _rm_suffix(".oca", configurator, variables["addon.name"] + "/README.rst.oca")
+    post_render_readme(configurator, addon_name=variables["addon.name"])
+    oca = (
+        variables["addon.oca"]
+        if isinstance(variables["addon.oca"], bool)
+        else to_boolean(False, False, variables["addon.oca"])
```
Comment (adrienpeiffer) [inline bobtemplates/odoo/hooks.py:216]: It's seems to_boolean is not executed on variabes. It's why a added this part of code

**Comment by adrienpeiffer** [comment]
Finally green :rocket:

File: bobtemplates/odoo/hooks.py [hunk 2]
```diff
@@ -86,6 +87,25 @@ def _rm_suffix(suffix, configurator, path):
     os.rename(path, path[: -len(suffix)])
 
 
+def _get_oca_readme_fragments(configurator, addon_name):
+    url = (
```
Comment (rousseldenis) [inline bobtemplates/odoo/hooks.py:91]: Maybe this as global variable to be quite visible?

**Review by rousseldenis — APPROVED** [review]
LGTM.

Just one question @adrienpeiffer

File: bobtemplates/odoo/hooks.py (hunk 2 above)
Comment (adrienpeiffer) [reply bobtemplates/odoo/hooks.py:91]: Sure bro

Comment (adrienpeiffer) [reply bobtemplates/odoo/hooks.py]: Done !

**Comment by rousseldenis** [comment]
Great!

File: bobtemplates/odoo/addon/+addon.name+/README.rst.acsone.bob [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+=========
+
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* ACSONE SA/NV
```
Comment (adrienpeiffer) [inline bobtemplates/odoo/addon/+addon.name+/README.rst.acsone.bob:41]: Use anser to fill this part

**Comment by sbidoul** [comment]
I just submitted the comments I wrote yesterday, but it looks like you got the same idea. IMO we should not bother with the credit section at all...

**Comment by adrienpeiffer** [comment]
@sbidoul done :small_airplane:

**Review by sbidoul — APPROVED** [review]
LGTM (code review only). Feel free to merge and release :)

**Pull request merged**